test/module/shared_model/backend_proto/permissions_test.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "backend/protobuf/permissions.hpp"
+#include <gtest/gtest.h>
+#include <boost/range/algorithm/for_each.hpp>
```

Comment:
is clang applied?

Reply:
actually it was, dunno why that isn't handled